src/main/java/io/github/opencubicchunks/cubicchunks/chunk/graph/CubeTicketType.java

```diff
@@ -6,7 +6,7 @@
 import io.github.opencubicchunks.cubicchunks.mixin.access.common.TicketTypeAccess;
 import net.minecraft.server.level.TicketType;
 
-public class CCTicketType {
+public class CubeTicketType {
     public static final TicketType<CubePos> CCPLAYER = create("player", Comparator.comparingLong(CubePos::asLong));
     public static final TicketType<CubePos> CCFORCED = create("forced", Comparator.comparingLong(CubePos::asLong));
     public static final TicketType<CubePos> CCLIGHT = create("light", Comparator.comparingLong(CubePos::asLong));
```

Comment:
While we're here, maybe add an `_` to these constants? `CCPLAYER` -> `CC_PLAYER`?

Reply:
Fields will be for later. I would just remove the CC prefix from them.